apps/acrs/sections/summary-data-sections.js

```diff
@@ -37,22 +37,6 @@ module.exports = {
             `${req.sessionModel.get('user-email')}` :
             `${req.sessionModel.get('referral-email')}`;
         }
-      },
-      {
-        steps: '/your-address',
-        field: 'your-address-line-1'
-      },
-      {
-        steps: '/your-address',
-        field: 'your-address-line-2'
-      },
-      {
-        steps: '/your-address',
-        field: 'your-address-town-or-city'
-      },
-      {
-        steps: '/your-address',
-        field: 'your-address-postcode'
```

Comment:
I think we need to keep this in for the other fields.

Reply:
Not sure we need this as we have concat the address in one field in the summary

Reply:
For accessibility I reckon it is probably better to have individual links for addresses where we can (like where its a single address that isn't aggregated) so that the change links can send the user back with the correct field already in focus.

If you wouldn't mind leaving it in for now as its unrelated to ACRS-46 then we can ask UCD what they think

Reply:
ACRS-46 is making use of this summary page not sure which one you refer to

Reply:
Just the your-address... sections code

Reply:
changed